app/helpers/template_config/census_config.py

```diff
@@ -0,0 +1,174 @@
+from dataclasses import dataclass, field
+from typing import Iterable, Mapping, MutableMapping
+
+from flask_babel import lazy_gettext
+from flask_babel.speaklater import LazyString
+
+from app.helpers.template_config.link import Link
+from app.helpers.template_config.survey_config import SurveyConfig
+
+CENSUS_EN_BASE_URL = "https://census.gov.uk"
+CENSUS_CY_BASE_URL = "https://cyfrifiad.gov.uk"
+CENSUS_NIR_BASE_URL = f"{CENSUS_EN_BASE_URL}/ni"
```

Comment:
Now that these are in their own file they could be shortened to `EN_BASE_URL` etc.